tests/unittests/PartitionerTest.cpp

```diff
@@ -385,3 +388,78 @@ TEST_F(PartitionerTest, SelectRepFunc) {
   auto err = myPartitioner.Partition();
   EXPECT_FALSE(errToBool(std::move(err)));
 }
+
+template <BackendKind kind, glow::Kinded::Kind unsupportedOpKind>
```

Comment:
Could you add a comment about the purpose of the class and describe the meaning of its template parameters.